ci-operator/jobs/rhobs/observability-operator/rhobs-observability-operator-main-periodics.yaml

```diff
@@ -54,6 +58,9 @@ periodics:
         readOnly: true
     serviceAccountName: ci-operator
     volumes:
+    - name: aws-qe-obo
+      secret:
+        secretName: aws-qe-obo
```

Comment:
need `namespace: test-credentials` for the secret

Reply:
I added that to the config spec, but the `jobs` make target removes that.